services/CollectionService/src/Controller/CollectionController.php

```diff
@@ -97,9 +108,8 @@ public function create(Application $app, Request $request, $id) {
             $putHeaders = $responsePut->getHeaders();
             //Guzzle psr7 response objects are inmutable. So we have to make this an array and add directly
             $putHeaders['Link'] = array('<'.$responsePut->getBody().'>; rel="alternate"');
-            $return_uuid = (isset($existingUuid) ? $existingUuid : $newUuid);
```

Comment:
I changed the way I "discovered" or "created" the UUID to both use `$uuid`. 

So I just use that in the returned headers instead of checking between 2 variables.